src/app/common/directives/oc-carousel/oc-carousel.js

```diff
@@ -0,0 +1,31 @@
+angular.module('orderCloud')
+    .directive('ocCarousel', ocCarouselDirective)
+;
```

Comment:
We should have some error message display if there are no slides defined on the buyer and there are no slides being passed in to the  directive.

Checkout how I used `$exceptionHandler` to do this on ocReorder as an example